pkg/interceptors/interceptors.go

```diff
@@ -161,3 +170,52 @@ func UnmarshalParams(ip map[string]interface{}, p interface{}) error {
 	}
 	return nil
 }
+
+// ResolveURL returns the URL for the given core interceptor
+func ResolveURL(i *triggersv1.TriggerInterceptor) string {
+	// This is temporary until we implement #868
+	base := fmt.Sprintf("http://%s.%s.svc/", CoreInterceptorsHost, system.GetNamespace())
```

Comment:
Let's build this from a url.URL so we're confident that things are URL encoded properly - I'm paranoid about how weird values being injected here might interact (e.g. `http://example.com/my/other/path/.foo.svc/`)

(while I'm pretty sure that we're safe here since we're using const / system derived values, better to be safe).

Reply:
done.

Reply:
I think this suffers from the same issue, since the parse wouldn't be able to distinguish which pieces were meant for the host. Let's do something like this instead?

```go
func ResolveURL(i *triggersv1.TriggerInterceptor) *url.URL, error {
  path := ""
  switch {
    ...
  }
  return &url.URL{
    Scheme: "http",
    Host: fmt.Sprintf("%s.%s.svc", CoreInterceptorsHost, system.GetNamespace()),
    Path: path,
  }
}

Reply:
ahh ok, I think I get the specific case you were talking about above. Will make the change.

Reply:
done.